src/image-button/image-button.style.tsx

```diff
@@ -0,0 +1,61 @@
+import { MediaQuery } from "../media";
+import styled from "styled-components";
+import { Color } from "../color/color";
+import { DesignToken } from "src/design-token/design-token";
+import { TextStyleHelper } from "src/text/helper";
+
+export const Main = styled.button`
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    justify-content: center;
+    align-items: center;
+    padding: 1.5rem 1rem;
+    background-color: ${Color.Neutral[8]};
+    border: none;
+    border-radius: 0.5rem;
+    cursor: pointer;
+    width: 13rem;
+    transition: all 200ms ease;
+    h4 {
+        ${TextStyleHelper.getTextStyle("H4", "semibold")}
+    }
+
+    img {
+        width: 3.5rem;
+    }
+
+    ${MediaQuery.MaxWidth.tablet} {
+        padding: 1.5rem 0rem;
```

Comment:
i think 1rem left/right padding should remain on tablet and mobile. it looks quite awkward otherwise
![Screenshot 2023-05-18 at 5 34 45 PM](https://github.com/LifeSG/react-design-system/assets/16853062/bc7eaf1d-e59e-42e7-a5fa-ff8507b240c0)

Reply:
Clarified with UX, I have mistakenly added shadow box for mobile and tablet view,  now it's fixed.